### PR TITLE
Obtain timeline data and expose an API for same

### PR DESCRIPTION
**Pull request by risabhbaheti**

### CODE REVIEW

File: src/data-store.ts [hunk 1]
```diff
@@ -30,6 +32,9 @@ export const getCovidData = (placeName: string) => {
     let newPlaceName = countryNameApiMapping.has(placeName)
         ? countryNameApiMapping.get(placeName)
         : placeName;
+    if(!timeLineData.has(newPlaceName)) {
+        countryTimeLineCB(newPlaceName);
+    }
```
Comment (ashubham) [inline src/data-store.ts:37]: Do we need to check if this is a country before we call the callback ?

File: src/data-store.ts [hunk 2]
```diff
@@ -39,6 +44,19 @@ export const getDemographicsData = (placeName: string) => {
...
+export const getCovidDataForDate = (placeName: string,
+                                    date: string) => {
+    return timeLineData.has(placeName)
+        ? timeLineData.get(placeName).date
+        : {};
+};
+
+export const countryTimeLineCB  = (countryName) => {
+    getCountrySpecificData(countryCodeMapping.get(countryName))
+        .then()
```
Comment (ashubham) [inline src/data-store.ts:56]: empty .then not needed

File: src/data-store.ts (hunk 1 above)
Comment (risabhbaheti) [reply src/data-store.ts:37]: We will not no if it is a country or state. As the data for all states and US has been prepopulated we don't need to make the call. Also we don't need to make the call once the data has been populated

File: src/data-store.ts (hunk 2 above)
Comment (risabhbaheti) [reply src/data-store.ts]: Done

**Pull request merged**